test/integration_test.sh

```diff
@@ -19,6 +35,10 @@ curl -s localhost:8080/static2/ducks.jpg > test_output_static2_jpg
 curl -s localhost:8080/static2/giphy.gif > test_output_static2_gif
 curl -s localhost:8080/static2/mario.jpeg > test_output_static2_jpeg
 
+#proxy test
+curl -i -s localhost:8080/test > test_proxy_echo2
```

Comment:
It's unclear whether the spec asked for this proxy integration test to be included as part of the other integration test. Maybe you could define this in a separate file and call that script from this integration test? To modularize it a little more? idk